src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb

```diff
@@ -183,21 +183,30 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             Task.WaitAny(task1, task2)
         End Sub
 
-        Public Overrides Async Function AssertCompletionSession(Optional projectionsView As ITextView = Nothing) As Task
+        Public Overrides Function AssertCompletionSession(Optional projectionsView As ITextView = Nothing) As Task
             Dim view = If(projectionsView, TextView)
 
-            Await WaitForAsynchronousOperationsAsync()
             Dim session = GetExportedValue(Of IAsyncCompletionBroker)().GetSession(view)
             Assert.True(session IsNot Nothing, "AssertCompletionSession")
+            Return Task.FromResult(0)
         End Function
 
+        Public Overrides Sub AssertCompletionItemsDoNotContainAny(displayText As String())
+            Dim items = GetCompletionItems()
```

Comment:
❔ Why no assertion here? Applies to the following method as well.

Reply:
As soon as we get completion items (modern), the completion is ready.

AssertNoAsynchronousOperationsRunning checks for operation which are not interesting in the context of the modern completion.

---
In reply to: [314785918](https://github.com/dotnet/roslyn/pull/38021#discussion_r314785918) [](ancestors = 314785918)